From 26b898fcc9e8ff877d4057b86d8c345d5a3a46cc Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 12 Jan 2024 09:50:44 +0100
Subject: [PATCH 0848/1141] arm64: dts: st: add power supplies to lvds on
 stm32mp257f-dk board

LVDS need two power supplies: v1v8 & vddcore

Change-Id: Idb4831066e6debdf520d2c77f7026b0a28b39ae3
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/350856
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
index 4c3a6851b428..527544df573e 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -451,6 +451,8 @@ ltdc_ep1_out: endpoint@1 {
 
 &lvds {
 	status = "okay";
+	vdd-supply =  <&scmi_vddcore>;
+	vdda18-supply = <&scmi_v1v8>;
 	backlight = <&panel_lvds_backlight>;
 	ports {
 		#address-cells = <1>;
-- 
2.39.2

